src/ros_commo.py

```diff
@@ -169,7 +169,7 @@ def glance_at(self, face_id):
 	# Turn only the eyes towards the given target point.
 	# Coordinates: meters; x==forward, y==to Eva's left.
 	def gaze_at_point(self, x, y, z):
-		print "gaze at point: ", x, y, z
+		#print "gaze at point: ", x, y, z
 
```

Comment:
too fast? There must be a bug -- this typically gets updated very infrequently.  Why would the rate be fast?